win32ss/user/ntuser/menu.c

```diff
@@ -3023,7 +3023,14 @@ static BOOL FASTCALL MENU_ShowPopup(PWND pwndOwner, PMENU menu, UINT id, UINT fl
         if ((y - height) < monitor->rcMonitor.top || y >= monitor->rcMonitor.bottom)
             y = monitor->rcMonitor.bottom - height;
         else
-            y -= height;
+        {
```

Comment:
Changed, but added ```INT``` before first use of ```adjHgt``` on the line below. @HBelusca, Please verify that you are OK with this. Otherwise, we had ```adjHgt``` ending line 3027 and beginning 3028 which seemed wasteful. Thanks.

Reply:
I verified that what you committed in your 3rd commit (having declaration and assignment placed on the same line) compiles fine in both msvc2010 and gcc4.7.2. i verified on releases/0.4.14 ofc.

Reply:
Thanks for your testing and report, @JoachimHenze.